openmetadata-ui/src/main/resources/ui/src/components/EntityInfoDrawer/EntityInfoDrawer.component.tsx

```diff
@@ -60,7 +60,7 @@ const EntityInfoDrawer = ({
           'owner',
           'columns',
           'usageSummary',
-          'tableProfile',
+          'profile',
```

Comment:
We can remove this as we are taking profiler details from different API

Reply:
We are using the same API, just the name of the field has changed